File: c/01/main.c
#include <stdio.h>

#define MAX_LEN 25

int main(int argc, char* argv[])
{
    char name[MAX_LEN];

    printf("What is your name? ");
    scanf("%s", name);
    printf("Hello, %s, nice to meet you!\n", name);

    return 0;
}

File: go/01/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	fmt.Print("What is your name? ")
	name, _ := reader.ReadString('\n')
	name = strings.Replace(name, "\n", "", -1)
	fmt.Print("Hello, ", name, ", nice to meet you!\n")
}
